docs/guides/qiskit-addons-sqd-get-started.ipynb

```diff
@@ -0,0 +1,2012 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b8f25160-5130-4913-9330-648444d5c77f",
+   "metadata": {},
+   "source": [
+    "# Getting started with SQD\n",
+    "\n",
+    "This guide demonstrates a simple working example to get started with the `qiskit-addons-sqd` package. In this example, you can use SQD to obtain an approximation of the ground state of the $N_2$ molecule at equilibrium.\n",
```

Comment:
Is SQD an acronym?

Reply:
Yes, `Sample-based quantum diagonalization`. We should probably use that name in the h1 and page title, then define the acronym first time it's seen.